src/ctia/http/routes/bulk.clj

```diff
@@ -0,0 +1,162 @@
+(ns ctia.http.routes.bulk
+  (:require [compojure.api.sweet :refer :all]
+            [clojure.string :as str]
+            [ctia.flows.crud :as flows]
+            [ctia.domain.entities :as ent]
+            [ctia.schemas
+             [bulk :refer [BulkRefs NewBulk StoredBulk]]]
+            [ctim.schemas.common :as c]
+            [ctia.store :refer :all]
+            [ring.util.http-response :refer :all]
+            [schema.core :as s]
+            [clojure.tools.logging :as log]))
+
+(defn singular [k]
+  "remove the last s of a keyword see test for an example."
+  (-> k
+      name
+      (str/replace #"s$" "")
+      keyword))
```

Comment:
Maybe move this to a lib?  Not sure what it would be.  Perhaps ctia.lib.keyword.  Also move tests.  :)